crates/gpui/src/platform/windows/window.rs

```diff
@@ -1398,6 +1417,21 @@ impl PlatformWindow for WindowsWindow {
     // todo(windows)
     fn on_active_status_change(&self, callback: Box<dyn FnMut(bool)>) {
         self.inner.callbacks.borrow_mut().active_status_change = Some(callback);
+        let inner = self.inner.clone();
```

Comment:
What's going on here?

Reply:
* `set_cursor_style` is only called when the window is active
* The initial value of active status is `false`
* `active_status_change` is called when `WM_ACTIVATE` is occur
* `WM_ACTIVATE` is not called if the window is active at the time of creation

Since the state was as described above, the callback needed to be called once here. But calling it directly would have resulted in a `BorrowMutError`, so the call was delayed.

Reply:
In second thought, it is better to add `PlatformWindow::is_active` and call it when initialize active status, isn't it?

Reply:
This part of codes can be removed after #9664 merged

Reply:
https://github.com/zed-industries/zed/assets/6465609/aa757158-5ed6-4fcf-b67e-debca035351c

https://github.com/kazatsuyu/zed/commit/76a00b1ffa86b9cf43961ae78cb33d8e79f7b77f
In my env, rebase on main & comment out this, the cursor style is still not applied immediately after window creation.
When window is inactivated and activated, it works fine

Reply:
However, this can separate to "fix initial active status" PR, I'll remove this

Reply:
Oh, I tested, you are right. An easy fix is using `PostMessageW(.., WM_ACTIVATE, WA_ACTIVE, ...)`

Reply:
It is essentially same to this (both is delaying to call `active_state_change`)